### PR TITLE
Added 'Safety checks' section to docs and linked to it from add command.

### LINKED ISSUE #26: Add link to documentation on selfdestruct warning
When `add`ing a contract with `selfdestruct`, the CLI raises a warning explaining it is a security risk (see https://github.com/zeppelinos/zos-cli/pull/347). Add a link to the documentation in the message so users can go and check for more information. 

If the documentation on that is not already written, then we need to write it as part of this issue.

### PR DESCRIPTION
**Pull request by ajsantander**
Fixes #26

### CODE REVIEW

File: packages/docs/docs/docs/advanced.md [hunk 1]
```diff
@@ -138,6 +138,13 @@ data = encodeCall(
 
 await exampleToken.sendTransaction( {data, from: owner} );
 ```
+## Safety checks
+
+The ZeppelinOS CLI performs a series of safety checks in some of its commands with the purpose of strengthening the security of your application. For example, the `zos add` command can detect if a contract has a `constructor`, or contains usages of `selfdestruct` or `delegatecall`. Below is a list of checks made, along with the reasoning behind why they are considered to be security risks:
+
+* constructor check: Proxied contracts should use initializer funcitons instead of constructors. For more info, see "Initializers vs. constructors" in this page.
```
Comment (jcarpanelli) [inline packages/docs/docs/docs/advanced.md]: nit: _funcitons_ => _functions_

File: packages/docs/docs/docs/advanced.md [hunk 2]
```diff
@@ -138,6 +138,13 @@ data = encodeCall(
 
 await exampleToken.sendTransaction( {data, from: owner} );
 ```
+## Safety checks
+
+The ZeppelinOS CLI performs a series of safety checks in some of its commands with the purpose of strengthening the security of your application. For example, the `zos add` command can detect if a contract has a `constructor`, or contains usages of `selfdestruct` or `delegatecall`. Below is a list of checks made, along with the reasoning behind why they are considered to be security risks:
```
Comment (elopio) [inline packages/docs/docs/docs/advanced.md]: s/application/project

File: packages/docs/docs/docs/advanced.md [hunk 3]
```diff
@@ -138,6 +138,13 @@ data = encodeCall(
...
+
+* constructor check: Proxied contracts should use initializer functions instead of constructors. For more info, see "Initializers vs. constructors" in this page.
```
Comment (elopio) [inline packages/docs/docs/docs/advanced.md]: You can link here to the section, right? I'm not sure how to make a relative link, but I suppose it's possible.

File: packages/docs/docs/docs/advanced.md [hunk 4]
```diff
@@ -138,6 +138,13 @@ data = encodeCall(
...
+* selfdestruct check: Solidity's `selfdestruct` keyword ends the execution of a contract, destroys it, and sends all of its funds to a specified account. This is a very delicate action to take, and can expose a contract to vulnerabilities such as the [second Parity Wallet hack](https://blog.zeppelinos.org/parity-wallet-hack-reloaded/). It should be used with extreme care.
```
Comment (elopio) [inline packages/docs/docs/docs/advanced.md]: s/take/make, I think.

File: packages/docs/docs/docs/advanced.md [hunk 5]
```diff
@@ -138,6 +138,13 @@ data = encodeCall(
...
+* delegatecall check: The `delegatecall` keyword fully exposes a contract's state to a 3rd party contract. Such a contract (to which the delegatecall is made to) has complete control on the calling contract. It should only be used if you really know what you're doing.
```
Comment (elopio) [inline packages/docs/docs/docs/advanced.md]: this sentence is hard to read: (to which the delegatecall is made to). Why not just removing it?

File: packages/docs/docs/docs/advanced.md (hunk 4 above)
Comment (ajsantander) [reply packages/docs/docs/docs/advanced.md]: I believe "take" is better.

**Comment by ajsantander** [comment]
Thanks for the review. Resolved some of the issues raised and added the clarification about `--force`.

@spalladino 
IMPORTANT: I rebased on master and noticed that the warnings moved to ValidationLogger.js, so I moved them there, taking the liberty of making a little abstraction on the links.

Also, since I detected that there are more warnings than what we are documenting in the docs, I changed them to say "these are some examples of validations". I think that if we document them all, we will need to do maintenance in the future.

**Comment by spalladino** [comment]
@ajsantander ok on the `ValidationLogger` front! I'd try to make the links to a specific 2.0.0 tag on the docsite, but it's ok for the time being until we actually have it.

**Pull request merged**